prebid_cache_client/client_test.go

```diff
@@ -234,7 +234,7 @@ func TestStripCacheHostAndPath(t *testing.T) {
 	}
 	for _, test := range testInput {
 		cacheClient := NewClient(&http.Client{}, &inCacheURL, &test.inExtCacheURL, &metricsConf.DummyMetricsEngine{})
-		cHost, cPath := cacheClient.GetExtCacheData()
+		_, cHost, cPath := cacheClient.GetExtCacheData()
```

Comment:
Shouldn't we have some test cases with a non-nil schema? in exchange_test.go, we are just using a mock cache client rather than this one.

Reply:
It's really just acting as a passthrough, but I no argument about adding test coverage. Fixed.